app/models/user.rb

```diff
@@ -14,6 +14,8 @@ class User < ActiveRecord::Base
     :two_factor_authenticatable,
     :omniauthable,
     authentication_keys: [:email],
+    case_insensitive_keys: [],
+    strip_whitespace_keys: [],
```

Comment:
ah! didn't realize we were configuring that -- assumed Devise was doing it automatically because it was an `authentication_key`.

Will fix in config file.